dataactcore/models/baseInterface.py

```diff
@@ -157,3 +159,10 @@ def getNameFromDict(self, model, dictName, fieldName, fieldValue, idField):
                 return key
         # If not found, raise an exception
         raise ValueError("Value: " + str(fieldValue) + " not found in dict: " + str(dict))
+
+
+@contextmanager
+def databaseSession():
+    interface = BaseInterface()
+    yield interface.session
+    interface.close()
```

Comment:
We would need to redesign the interfaces to accommodate this usage.  By closing the connection after each use, any existing interfaces will hit errors because they share a connection.